### ISSUE TITLE
A problem when I install Brian2

### ISSUE THREAD

**Issue by hustyanghui** [brian-team/brian2#930]
Hello everyone,
       Recently I want to use anaconda3 and brian2 ,for reference:
            http://brian2.readthedocs.io/en/stable/introduction/install.html
       And I have installed Anaconda successfully!
      But when I want to install brian2 as the link suppose : conda install -c brian-team brian2
      There comes the  question.The result after is showed as follow:
       "
     Solving environment: failed

UnsatisfiableError: The following specifications were found to be in conflict:
  - brian2
  - numba
Use "conda info <package>" to see the dependencies for each package.

        "

So ,is there anyone else have meet this problem,would you like to give me some useful suggestions?Thanks very much!

**Comment by hustyanghui** [comment]
For supplement,I installed Anaconda in Linux

**Comment by mstimberg** [comment]
Hi. From the error message it is difficult to see what exactly is going wrong. Most likely the problem is that a package depends on numpy 1.14, while the current Brian2 package only supports numpy 1.13.
Your error message seems to state that there's a problem with the numba package, but that's odd given that it could downgrade numba to the numpy 1.13 version.
I'd try to manually downgrade numpy to version 1.13:
```
conda install numpy==1.13
```
If that doesn't work because some package requires the most recent numpy version, you can always create a conda environment (see https://conda.io/docs/user-guide/tasks/manage-environments.html) just for Brian. Try:
```
conda create -n brian2 -c brian-team python=3 brian2 matplotlib
```
(you can add other packages that you want to use at the end, e.g. `notebook` for the jupyter notebook)

Then, you can switch to this environment with
```
source activate brian2
```

**Comment by mstimberg** [comment]
This should be fix with the latest release (currently: 2.1.3.1), please re-open this issue if you still have problems.
